### ISSUE TITLE
Parse error with LANGUAGE pragma

### ISSUE THREAD

**Issue by ndmitchell** [haskell-suite/haskell-src-exts#217]
Given:

    {-# LANGUAGE
    RecordWildCards
     #-}
    main = print "hello"

This works in GHC, but raises a parse error in HSE. Note that `#-}` line is indented by one space - that's important or it's a parse error in GHC too. Originally from https://github.com/ndmitchell/hlint/issues/125

**Comment by mpickering** [comment]
This is the same as #228 

The lexer injects a semi colon before `RecordWildCards` as it is in the same column as the `{-#` above.

**Comment by pjonsson** [comment]
I don't think it's the same problem. The lexer should insert a semicolon in this case but our parser doesn't expect to find a semicolon in the middle of a language pragma. GHC has something similar in the grammar for deprecations to this:
```Haskell
> conids    :: { ([Name L],[S]) }
>          : conids ',' conid                  { (fst $1 ++ [$3], snd $1 ++ [$2]) }
>          | optsemis conid                    { ([$2],[]) }
```

**Comment by pjonsson** [comment]
Fixed with 52d1967210836b3d43b28581836addc1ecb9254c.
